airgun/views/subscription.py

```diff
@@ -21,6 +21,16 @@
 )
 
 
+def wait_for_spinner(widget):
+    """Wait for any spinner to disappear"""
+    wait_for(
+        lambda: not widget.browser.elements("//div[contains(@class, 'spinner')]"),
```

Comment:
Oh, basically the key question - since we have multiple pages with such spinner, maybe it's not worth separate helper which has to be manually called each time but rather worth adding new check to ensure_page_safe? That would ensure spinner isn't present in all possible cases (way more than covered ones) and won't require extra lines. Thoughts?

Reply:
@oshtaier and your thoughts, pls?

Reply:
It depends on the impact on performance. If we talk about milliseconds for that validation, so it is not a problem. Anyway, can we override `ensure_page_safe` or similar idea just for that module? I meant, I don't like that we have more than one (actually at least 3) methods to wait for different things. It is just bad style